ddl/index.go

```diff
@@ -324,43 +326,100 @@ func (d *ddl) onDropIndex(t *meta.Meta, job *model.Job) error {
 	return errors.Trace(err)
 }
 
-func (d *ddl) fetchRowColVals(txn kv.Transaction, t table.Table, batchOpInfo *indexBatchOpInfo, seekHandle int64) error {
-	cols := t.Cols()
-	idxInfo := batchOpInfo.tblIndex.Meta()
+func isFinished(limit, input int64) bool {
```

Comment:
The function name is too generic.
make it as a method of `handleInfo` is better.